core-concepts/wallets/veworld/user-guide/settings.md

```diff
@@ -0,0 +1,174 @@
+---
+description: How to customize your wallet
+layout:
+  title:
+    visible: true
+  description:
+    visible: true
+  tableOfContents:
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: true
+---
+
+# Settings
+
+{% tabs %}
+
+{% tab title="Mobile" %} 
+
+## General
+
+**Conversion Currency**
+
+You can choose amongst the supported FIAT currencies (EUR, USD)
+
+**Theme**
+
+You can choose dark or light theme, or to follow your system configuration.
+
+**Reset**
+
+You can hard reset your wallet, and clear all your local data.
+This action cannot be reverted.
```

Comment:
I would add a disclaimer here, it must be clear that the action can incur in a loss of money

Reply:
Done! @victhorbi